DigitalOffice/Services/ProjectService/Commands/EditProjectByIdCommand.cs

```diff
@@ -37,9 +37,9 @@ public Guid Execute(Guid projectId, EditProjectRequest request)
             }
 
             request.Id = projectId;
-            
+
             validator.ValidateAndThrow(request);
-            
+
             var dbProject = mapper.Map(request);
 
             return repository.EditProjectById(dbProject);
```

Comment:
This indentation looks dumb. There are not so many lines to put an enter between them.
That's my personal opinion though

Reply:
Fixed